packages/contracts/contracts/THUSDToken.sol

```diff
@@ -89,6 +89,7 @@ contract THUSDToken is Ownable, CheckContract, ITHUSDToken {
         _CACHED_CHAIN_ID = _chainID();
         _CACHED_DOMAIN_SEPARATOR = _buildDomainSeparator(_TYPE_HASH, hashedName, hashedVersion);
         governanceTimeDelay = _governanceTimeDelay;
+        require(governanceTimeDelay <= 30 weeks, "Governance delay is too big");
```

Comment:
What's the rationale for `30 weeks`? I know we want to cap the value that can be passed, but 30 weeks is a long time 😅 . 

Or is there some calc between milliseconds/seconds confusion that makes this value a good one to use as a sanity check?

Reply:
That is 100% magic number, I remember that initially we discuss 2 month as value for delay. So I wanted to put something bigger than 8-10 weeks in several times.